packages/fxa-content-server/app/scripts/views/mixins/brand-messaging-mixin.js

```diff
@@ -43,13 +43,24 @@ const Mixin = {
       this.logFlowEvent(`brand-messaging-${this.mode}-view`, this.viewName);
 
       // hack add some padding to prevent overlap
-      setTimeout(() => {
-        console.log('!!! adding padding2');
-        document.body.classList.add('brand-messaging');
-      }, 300);
+      // setTimeout(() => {
+      //   document.body.classList.add('brand-messaging');
+      //   document.querySelector;
+      // }, 0);
```

Comment:
Should this entire bit be removed since it's commented out?

Not sure if this is related, and I can't seem to reproduce reliably, but playing around with the window resizing got me in a weird state with two banners at least once:
<img width="500" alt="image" src="https://github.com/mozilla/fxa/assets/13018240/fe146577-b2ae-46eb-979e-e41f4cdebbcf">

Reply:
I saw the same once, but haven't been able to repro since! 🤔

Reply:
I can't reproduce and just pushed up new changes. Mind trying again?